.enb/lib/techs.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
maybe we can extend this file and publish as npm module with all the deps bundled?